api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -238,6 +238,52 @@ message HttpConnectionManager {
     google.protobuf.BoolValue enabled = 3;
   }
 
+  // Transformations that apply to path headers. The policy to apply these
+  // transformations can be tuned by the configuration field. Transformations
+  // can be applied internally only, so that the normalized header is used in
+  // matching, routing, and HTTP filters while the original path header is
+  // forwarded upstream, or additionally apply to the forwarded *:path* header.
+  message PathNormalizationOptions {
+    // Defines a policy to apply normalization.  Path rewrites :ref:`regex_rewrite
+    // <envoy_api_field_route.RouteAction.regex_rewrite>` or :ref:`prefix_rewrite
+    // <envoy_api_field_route.RouteAction.prefix_rewrite>` will apply to the
+    // *:path* header destined for the upstream.
```

Comment:
Why only mention these two?

Reply:
Those are the only router transformations, but that may change in the future so I will change the language.

I mentioned that in this comment, it's only relevant for `INTERNAL`:
> // Normalizations only apply internally. The original *:path* header will
      // be sent upstream *unless* the path has been mutated by a filter.